contracts/shiden34/Cargo.toml

```diff
@@ -1,41 +1,33 @@
 [package]
 name = "shiden34"
-version = "0.2.0"
-authors = ["Stake Technologies <devops@stake.co.jp>"]
+version = "1.0.0"
+authors = ["Astar builder"]
 edition = "2021"
 
 [dependencies]
-ink_primitives = { version = "3.4.0", default-features = false }
-ink_metadata = { version = "3.4.0", default-features = false, features = ["derive"], optional = true }
-ink_env = { version = "3.4.0", default-features = false }
-ink_storage = { version = "3.4.0", default-features = false }
-ink_lang = { version = "3.4.0", default-features = false }
-ink_prelude = { version = "3.4.0", default-features = false }
+ink = { version = "~4.0.0", default-features = false}
 
 scale = { package = "parity-scale-codec", version = "3", default-features = false, features = ["derive"] }
-scale-info = { version = "2", default-features = false, features = ["derive"], optional = true }
+scale-info = { version = "2.3", default-features = false, features = ["derive"], optional = true }
 
-openbrush = { tag = "v2.3.0", git = "https://github.com/Supercolony-net/openbrush-contracts", default-features = false, features = ["ownable", "psp34", "reentrancy_guard"] }
+openbrush = { tag = "3.0.0", git = "https://github.com/727-Ventures/openbrush-contracts", default-features = false, features = ["psp34", "ownable"] }
 payable_mint_pkg = { path = "../../logics", default-features = false }
 
-
 [lib]
 name = "shiden34"
 path = "lib.rs"
 crate-type = [
-	# Used for normal contract Wasm blobs.
-	"cdylib",
+    # Used for normal contract Wasm blobs.
+    "cdylib",
 ]
```

Comment:
We can remove the `name` & `crate-type`, they are not needed anymore.
https://use.ink/faq/migrating-from-ink-3-to-4/#removal-of-libcrate-type-and-libname-from-contract-manifest

Reply:
ok